Clinical-Genomics/cgweb
# removed functionality for trending

How to test:
0. install on clinical-db stage
1. log in

Expected result:
No trending views should be available
